org.gimp.GIMP.json

```diff
@@ -716,8 +716,7 @@
                 {
                     "type": "git",
                     "url": "https://gitlab.gnome.org/GNOME/gimp.git",
-                    "tag": "GIMP_2_10_30",
-                    "commit": "cee406b593fb561efebc3664bdc0790fa45b9c47"
+                    "branch": "gimp-2-10"
```

Comment:
Any reason for the switch from `tag`+`commit` to `branch`?

Reply:
It's only temporary since we don't have a `GIMP_2_10_32` tag yet but we want to test with latest code **before** GIMP 2.10.32 is actually released (to detect problems early). So this is what I usually do once I start testing for a release: the first commit switch to `branch` to test the `HEAD` of the branch. Then the last commit switch back to `tag` once we have a tag published.

Reply:
Ah, that makes sense. Thanks for the explanation!